### PR TITLE
[Term Entry] Dart Queue .clear() #4457

### PR DESCRIPTION
**Pull request by NixDi**
### Description

- Added a clear.md file. 
- Added content related to Dart Queue .clear() method.
- Closes issue #4457 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/dart/concepts/queue/terms/clear/clear.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Tags:
+  - 'Methods'
+  - 'Dart'
+  - 'Classes'
+  - 'Queues'
+CatalogContent:
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+In [`Dart`](<[url](https://www.codecademy.com/resources/docs/dart)>), the **`.clear()`** method in [`Queue`](<[url](https://www.codecademy.com/resources/docs/dart/queue)>) removes all elements in the queue. The size of the queue becomes zero.
```
Comment (ishg-153) [inline content/dart/concepts/queue/terms/clear/clear.md:16]:
```suggestion

In Dart, the **`.clear()`** method in Queue removes all elements in the queue. The size of the queue becomes zero.
```
It doesn't make sense to add those links as it is clear that this entry is under dart topic and queue concept.

File: content/dart/concepts/queue/terms/clear/clear.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+A brief description of the features of the `.clear()` method:
+- removes all elements in the queue;
```
Comment (ishg-153) [inline content/dart/concepts/queue/terms/clear/clear.md:19]: Remove this as it is redundant.

File: content/dart/concepts/queue/terms/clear/clear.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- the method doesn't return anything (`void`);
+- the method doesn't take any arguments.
```
Comment (ishg-153) [inline content/dart/concepts/queue/terms/clear/clear.md:21]: Instead of writing this as points make it into a sentence.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@NixDi I have suggested some changes. Please make those and let me know once done.

**Comment by NixDi** [comment]
@ishg-153 Thanks for your reply. I have made the changes as requested, please check.

**Review by ishg-153 — APPROVED** [review]
@NixDi I apologize for the delayed response. This entry is ready for a second review!

**Review by letsbuildanything — APPROVED** [review]
Looks good to be merged! 🚀

**Pull request merged**